pkg/api/customization/cluster/validator.go

```diff
@@ -48,18 +53,33 @@ func (v *Validator) Validator(request *types.APIContext, schema *types.Schema, d
 		return err
 	}
 
-	if err := validateScheduledClusterScan(&spec); err != nil {
-		return err
+	if err := v.validateScheduledClusterScan(&spec); err != nil {
+		return httperror.NewAPIError(httperror.InvalidBodyContent, err.Error())
```

Comment:
If we are wrapping the error, would the message have duplicated http code? since you are wrapping one http error with another http error that could possibly be returned in `validateScheduledClusterScan` method.